src/app/containers/OptimizelyPageViewTracking/index.jsx

```diff
@@ -1,6 +1,7 @@
 import { useState, useContext, useEffect } from 'react';
 import { OptimizelyContext } from '@optimizely/react-sdk';
 import { RequestContext } from '#contexts/RequestContext';
+import useScrollDepth from '#hooks/useScrollDepth';
```

Comment:
Might be better to rename this to `useOptimizelyScrollDepth` or something similar? Just because it's tightly coupled with optimized, it could be confusing that this could be used in a generic way

Reply:
Done 👍